src/daily-quest/daily-quest.service.ts

```diff
@@ -71,9 +78,9 @@ export default class DailyQuestService {
     userId: string,
     form: CreateDailyQuestFormType
   ): Promise<DailyQuestModel> => {
-    await validateGoalExists(userId, form);
-
     try {
+      await validateDailyQuestForCreate(userId, form);
```

Comment:
I think we can omit For word and meaning will be the same - validateDailyQuestCreate
Same with update

Reply:
Than probably: `validateCreateDailyQuest` or `validateDailyQuestCreating`.
What do you think?

Reply:
validateCreateDailyQuest

Reply:
Done!